.github/workflows/integration.yml

```diff
@@ -11,7 +11,12 @@ jobs:
       options: --gpus all
     runs-on: [self-hosted, linux, x64, common]
     steps:
-    - uses: actions/checkout@v2
+    # checkout the pull request branch
+    - uses: actions/checkout@v2.1.0
```

Comment:
It would be nice to keep `actions/checkout@v2` here.

Reply:
thanks!